packages/playht/src/grpc-client/client.ts

```diff
@@ -49,13 +49,28 @@ export class Client {
   }
 
   private async getLease() {
-    const response = await fetch(`${this.apiUrl}/v2/leases`, {
+    let response = await fetch(`${this.apiUrl}/v2/leases`, {
       method: 'POST',
       headers: this.apiHeaders,
     });
+    // Retry once with 5 second wait.
     if (!response.ok) {
-      const responseJson = await response.json();
-      throw new Error(`Failed to get lease: ${response.status} ${response.statusText} - ${responseJson.error_message}`);
+      await new Promise(r => setTimeout(r, 5000));
```

Comment:
5 second could be a lot of silence. Can we do a second or make it configurable

Reply:
I was thinking that refreshLease() was only called by the lease timer, but I see that it's also called on stream creation. I'll revert this retry to fail-fast and instead rely on the retry via timer.

Reply:
I think I'll also make a more insistent retry timer.  Have it retry every 30 seconds if it's failing.